ci/docker/install/ubuntu_r.sh

```diff
@@ -20,17 +20,31 @@
 # build and install are separated so changes to build don't invalidate
 # the whole docker cache for the image
 
+# Important Maintenance Instructions:
```

Comment:
This is probably unintended, but the change to this file causes CI to occasionally fail.  The keyserver for this gpg key fails approx 1/10 requests.  Is there a good reason we reverted @larroy change which manually installs the key?   It seemed like a reasonable fix for the flaky gpg server issue.  Any objection to me restoring the manual key add?